src/featureFlags.ts

```diff
@@ -1,5 +1,20 @@
-export type FeatureFlag = 'useNewConnectionForm';
-export type FeatureFlags = Record<FeatureFlag, boolean>;
-export const getFeatureFlags = (): FeatureFlags => ({
+const FEATURE_FLAGS = {
   useNewConnectionForm: `${process.env.MDB_USE_NEW_CONNECTION_FORM}` === 'true',
-});
+};
+
+export type FeatureFlag = keyof typeof FEATURE_FLAGS;
+
+export const getFeatureFlag = (flag: FeatureFlag) => {
+  if (typeof window === 'object') {
+    return (window as any).MDB_FEATURE_FLAGS?.[flag];
```

Comment:
Might be easier to use `globalThis` instead of `window` if some tests run in e.g. a plain Node.js environemnt?

Reply:
the idea here was to fetch the flags from an this `MDB_FEATURE_FLAGS` object attached on window when in html/react world. For the rest, I believe the return below would work just fine, right?

Reply:
I guess I meant to say that in plan node environment I was relying on the return below to do the trick.

Reply:
Sure, whatever makes more sense to you :)